plugins/commands/box/command/update.rb

```diff
@@ -37,6 +37,10 @@ def execute
               options[:force] = f
             end
 
+            o.on("-i", "--ignore-failures", "Ignores box and attempts to continue boxes in environment") do |i|
+              options[:ignore] = i
```

Comment:
I'm flexible on this name. Right now it only ignores boxes with no metadata, (i.e. boxes added manually, likely from packer builds).